packages/components/src/SVGIcon/SVGIcon.stories.tsx

```diff
@@ -1,9 +1,9 @@
+import * as ColorTokens from "@chanzuckerberg/eds-tokens/ts/colors";
```

Comment:
This is exactly the same as `js/colors.js`, but avoids the error
```
Could not find a declaration file for module 'module-name'. '/path/to/module-name.js' implicitly has an 'any' type
```

I think we need to keep the `js` one around still, for flexible use